src/app/dashboard/dashboard.html

```diff
@@ -1,33 +1,26 @@
+<<<<<<< HEAD
+<div id="dashboardContainer">
+
+	<header id="dashboardHeader">
+			<nav>
+				<ul>
+					<li>
+						<img src="" />
```

Comment:
not quite sure what the  `<img src="">` is for.  could we remove it if we are not using it